### ISSUE TITLE
cd autocomplete in a non-existing directory breaks

### ISSUE THREAD

**Issue by darthdeus** [fish-shell/fish-shell#423]
to reproduce:

```
mkdir foo
cd foo
```

in another terminal do `rmdir foo`

and in the first one try to `cd ~/<TAB>`, which results in the following error

```
The directory '/Users/darth/foo' does not exist                                          @
/usr/local/Cellar/fishfish/HEAD/share/fish/functions/__fish_complete_cd.fish (line 53): 	builtin cd $wd
                                                                                                ^
in function '__fish_complete_cd',
	called on standard input,

in command substitution
	called on standard input,
```

**Comment by faho** [comment]
I can't reproduce this, might have been fixed by cb86b8f8fcf64308b6f4599281c6197c547fa32c.

**Comment by ridiculousfish** [comment]
I can't repro either, let's close. Thanks!
